src/Vixen.Common/Controls/TimeLineControl/Grid_Mouse.cs

```diff
@@ -1018,7 +1018,7 @@ private void MouseMove_HResizing(Point gridLocation, Point delta)
 					if (AltPressed && adjoiningElements.Count() == 0)
 					{
 						// Get the Elements that are following the Selected Element
-						List<Element> followElements = Rows.First().GetFollowersOfElement(SelectedElements.First());
+						List<Element> followElements = ActiveRow.GetFollowersOfElement(SelectedElements.First());

```

Comment:
Same as above.

Reply:
Changing the asignment of ActiveRow may have unforseen ramifications. I instead changed to use your recommendation.